brigade/plugins/tasks/files/write.py

```diff
@@ -0,0 +1,48 @@
+import difflib
+import os
+
+from brigade.core.task import Result
+
+
+def read_file(file):
+    if not os.path.exists(file):
+        return []
+    with open(file, "r") as f:
+        return f.read().splitlines()
+
+
+def generate_diff(filename, content, append):
```

Comment:
Should `read_file` and `generate_file` be prefixed with a `_`?

If we decide to allow multiple tasks within the same file we might want to collect all of the tasks dynamically, then we could just exclude _read_file and _generate_diff.

Reply:
you are right, will mark them as private.